tock/tock/tests/js/integration/timecard.test.js

```diff
@@ -81,7 +81,86 @@ describe("Timecard", () => {
         text: "0.6",
       });
     });
-  })
+  });
+
+  describe("handles weekly billing", () => {
+    beforeEach(() => page.goto(`${baseUrl}/reporting_period/2015-03-30/`));
+
+    test("hides hours input and shows project allocation element", async () => {
+      // https://github.com/18F/tock/issues/848
+      await page.evaluate(() => {
+        document.querySelector("#id_timecardobjects-0-hours_spent").value = ''
+      });
+
+      // @TODO: Seed database with a weekly billing project and know the ID for this test
```

Comment:
**question:** It looks like this is already done in the test data, do we still need this TODO comment?